eth-denver/welcome.md

```diff
@@ -0,0 +1,117 @@
+```sh
+# TODO bashjitsu
+```
+This should get you ready to begin solving the riddle! Once you think you have the answer you can submit it with a commit reveal scheme. This means that you first submit a hash of your address and your answer so that no one can see what you think the answer is. Following this is the plain text submssion of your answer. At this point the app will hash your submission with your address to make sure you have already commited it. This prevents someone else from submitting your answer by watching the mempool of pending transactions.
+
+Your commit command should look like the following:
+```sh
+ebcli scavenge commit "your answer" --from nickname
+```
+This command will hash it on your behalf and submit the transaction after asking you to sign with the key you generated earlier. Once submitted you can confirm it was successful by taking the resulting <txhash> (looks like `TODO`) and querying the status of the tx as follows:
+```sh
+ebcli query tx <txhash>
+```
+You should be able to see a series of events which were triggered when this transaction was sent. If it was successful you can proceed to reveal your answer and claim your reward:
+```sh
+ebcli scavenge reveal "your answer" --from nickname
+```
+This command will build the transaction and submit it after you sign with your key. To check whether the transaction was successful you can run a similar command as we previously did to check the transaction status:
+```sh
+ebcli query tx <txhash>
+```
+You could also query the scavange directly to see if your name shows up as the solver:
+```sh
+ebcli query scavenge show <solutionHash>
+```
+If you see your name you should be able to confirm that you received the NFT reward as too by using the following command:
+```sh
+ebcli query nft balance $(ebcli keys show nickname -a)
+```
+Now that you've earned the NFT you can transfer it back to your Ethereum Address on the xDai chain with the following peggy command:
+```
+#TODO ebcli peggy transferNFT
+```
+Congratulations, you've just won XP and/or an NFT on the Eth Denver Cosmos Burner Chain! This chain will shut down after it's purpose at this event but the assets you earned will live on to be used in the DAOStack community voting pool and stay with you on the much less ephemeral xDai and Ethereum blockchains!
+
+If you enjoyed this Tutorial feel free to share to others who may also enjoy it! If you had any issues we'd love to hear about them! This tutorial is hosted on github where you can make a new issue and describe the difficulties you were experiencing. If you want help during the hackathon just look for someone at the Cosmos booth!
+
+If you want to stay up to date with me consider following my twitter, github and/or medium.
```

Comment:
You may want to add links here and introduce yourself somewhere in this tutorial, since otherwise it'll be anonymous on the tutorials site.